projects/plugins/jetpack/modules/search/instant-search/lib/hooks/use-photon.js

```diff
@@ -27,17 +27,21 @@ function stripQueryString( url ) {
  */
 export function usePhoton( initialSrc, width, height, isPhotonEnabled = true ) {
 	const [ src, setSrc ] = useState( null );
+	const initialSrcWithoutQueryString = stripQueryString( initialSrc );
+
+	// Photon does not support SVGs
+	const isSVG = initialSrcWithoutQueryString?.substr( -4 ).toLowerCase() === '.svg';
```

Comment:
> We only fetch, resize, and serve gif, png, and jpg images from servers that listen on port 80 for HTTP and port 443 for HTTPS. This is about 99.99% of the web servers in the world. 

@see limitations https://developer.wordpress.com/docs/photon/

Could we maybe white list the extension rather than testing only for SVG?

Reply:
That's a good idea 👍 We might run into the same thing with other formats otherwise. I don't think `.webp` is supported either.

Reply:
Fixed in 09f51ff 👍